custom_components/ecowitt/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "ecowitt",
   "name": "Ecowitt Weather Station",
+  "version": "0.5",
```

Comment:
Should this have three version components, just to be safe? (0.5.0)

https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes/#versions

> The manifest.json file now has added support for a version key. The version should be a string with a major, minor and patch version. For example, "1.0.0".

Reply:
@esev, the short answer is no.

Could you please read the page, what you referenced, from the top? You will find the answer why the version key is necessary.

Many components uses nowadays the 2021.X or 2021.X.Y numbering as well.

Reply:
I probably phrased my question incorrectly. I edited above to clarify what I meant `"version": "0.5.0"`. Thanks for catching that.

I'm thinking since the HA developers specifically asked for the version string to contain three parts, it might be more future-proof if three parts are used in this PR.

Reply:
That is an example only. The whole point is to have version control.

A developer might use other things as well, like 0.1-alfa.